spec/views/application_records/csv_export_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'csv'
+include RSpecHtmlMatchers
+
+describe 'application_records/csv_export.csv.erb' do
+  before :each do
+    @id = 1
```

Comment:
Seeing hardcoded IDs makes alarms go off in my head. It's fine here!

But perhaps a quick comment explaining the structure of this test and the motivation thereof (since it's the only one of its kind).